packs/elastic/README.md

```diff
@@ -0,0 +1,170 @@
+**all_indices** | Set to `true` to operate on all indices in a cluster. | This option overrides other filtering parameters except **exclude**.
+
+### Snapshot selection only
+
+Parameter | Description | Details
+------------ | ------------ | ------------
+**snapshot** | Comma separated string of snapshot names to be included into the operation. | Snapshots added with this option **will not** be filtered by any of the other snapshot selection parameters.
+**all_snapshots** | Set to `true` to operate on all snapshots in a cluster. | This option overrides other filtering parameters except **exclude**.
+**repository** | Provides the repository name for snapshot operations (**required**).
+
+## Search actions
+
+Search actions perform a specified query in Elasticsearch. There are two search actions available: **search.q** and **search.body**. The first one takes a query string (given in lucene syntax), while the former allows to perform more sophisticated searches using Elasticsearch query DSL.
+
+Both search actions use the same *common parameters* as curator based actions.
+
+### search.q specific parameters
+
+This action is enhanced with *index selection* parameters to simplify indices matching.
+
+Parameter | Description
+------------ | ------------
+**q** | Query in the Lucene query string syntax (**required**).
+**df** | The default field to use when no field prefix is defined within the query.
+**default_operator** | The default operator to be used, can be AND or OR. Defaults to OR.
+**from** | The starting from index of the hits to return. Defaults to 0.
+**size** | The number of hits to return. Defaults to 10.
+**pretty** | Set to `true` to pretty print JSON response.
+
+### search.body specific parameters
+
+Parameter | Description 
+------------ | ------------
+**body** | The search definition using the Query DSL (**required**). 
+**indices** | A comma-separated list of index names to search. Defaults to `"_all"`.
+**from** | The starting from index of the hits to return. Defaults to 0.
+**size** | The number of hits to return. Defaults to 10.
+**pretty** | Set to `true` to pretty print JSON response.
+
+## Usage and examples
+
+Performing *curator operations* on indices or snapshots **at least one** filtering parameter must be specified. That's a generic rule applied to all of curator actions except *show*,  *show* will act on all indices or snapshots if there aren't any other filtering options.
+
+Now let's have at a few invocation examples.
+
+### Show and deleting indices
+
+* Show indices older than 2 days:
+```
+st2 run elastic.show.indices host=elk older_than=2 timestring=%Y.%m.%d
+```
+Shows this on my node:
+```json
+{
+    "result": null, 
+    "exit_code": 0, 
+    "stderr": "", 
+    "stdout": "logstash-2015.05.02
+logstash-2015.05.04
+"
+}
+```
+* Delete all indices matching *^logstash.\**:
+
+```
+st2 run elastic.delete.indices host=elk prefix=logstash
+```
+
+### Snapshot operations
```

Comment:
Yeah. It's possible, *show.snapshots* then *delete.snapshots*, but only these two operate on snapshots.

The `elastic.snapshot` action actually operates on indices, you only give the repository name to store snapshot, and indices selection options, and as the result you'll get a **curator-%%%blabla_date_time** of **selected** indices.